packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -323,37 +394,82 @@ export const Audio = () => {
       uri: m3u8
     }
   }
+
+  let nextSource
+  if (nextOfflineTrackUri) {
+    nextSource = {
+      type: TrackType.Default,
+      uri: nextOfflineTrackUri
+    }
+    // TODO: remove feature flag - https://github.com/AudiusProject/audius-client/pull/2147
+  } else if (isStreamMp3Enabled && nextStreamingUri) {
+    nextSource = {
+      type: TrackType.Default,
+      uri: nextStreamingUri
+    }
+  } else if (nextM3u8) {
+    nextSource = {
+      type: TrackType.HLS,
+      uri: nextM3u8
+    }
```

Comment:
Is the player working well for both segments and mp3 streaming?

Reply:
Yes, last that I checked both were working